### ISSUE TITLE
The `multiple` option does not work

### ISSUE THREAD

**Issue by wickedest** [cb1kenobi/cli-kit#52]
Providing `multiple` to an option should produce an array.  It does not.  E.g. given,
```js
{
    options: {
        '--filter=<pattern>': { type: 'string', desc: 'Filter', multiple: true }
    }
}
```
The command action is only ever given one `filter`, even if multiple `filter` are provided from the CLI.

**Comment by cb1kenobi** [comment]
Verified this is not working as advertised. When `multiple` is `true`, it does initialize the value to an array, but then overrides the array with the string value instead of pushing the string value into the array.

Furthermore, since the option is required and it has been initialized to an empty array, the empty array is truthy causing the required check to pass. So, 2 bugs in 1. Fix is on its way.

**Comment by cb1kenobi** [comment]
Fixed and release with v0.12.1.
